clients/projectshare/src/test/ProjectShareClient.test.ts

```diff
@@ -154,7 +425,10 @@ describe("ProjectShareClient (#integration)", () => {
   }
 
   it("should be able to CRUD custom properties", async () => {
-    const firstImageFile = (await projectShareClient.getFiles(requestContext, projectId, new ProjectShareFileQuery().startsWithPathAndNameLike(projectId, "360-Images/2A", "2A_v0410470")))[0];
+    let FoldersInRootFolder: ProjectShareFolder[] = await projectShareClient.getFolders(requestContext, projectId, new ProjectShareFolderQuery().inRootFolder(projectId));
+    await CreateFoldersWithFiles(projectShareClient, requestContext, projectId, FoldersInRootFolder[0].wsgId)

```

Comment:
Is it really worth it to use CreateFoldersWithFiles in every test?

Reply:
in this test CreateFoldersWithFiles is necessary